tools/rapids-wheel-ctk-name-gen

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# A utility script that generates CUDA suffix and wheel name for wheel-related workflows
+# Positional Arguments:
+#   1) ctk-tag
+#   2) package name
+set -eu -o pipefail
+
+if [ -z "$1" ] || [ -z "$2" ]; then
+  rapids-echo-stderr "Must specify both input arguments: CTK_TAG and PACKAGE_NAME"
+  exit 1
+fi
+
+ctk_tag="$1"
+ctk_major=$(echo "$ctk_tag" | cut -d'.' -f1)
+cuda_suffix="-cu${ctk_major}"
+wheelname="$2_cu${ctk_major}"
+
+echo "RAPIDS_PY_WHEEL_CUDA_SUFFIX=${cuda_suffix}" | tee --append "${GITHUB_ENV}" "${GITHUB_OUTPUT}"
+echo "RAPIDS_PY_WHEEL_NAME=${wheelname}" | tee --append "${GITHUB_ENV}" "${GITHUB_OUTPUT}"
```

Comment:
This is not that useful if someone wants to use this outside the actions workflow.

Reply:
I figured these are just `gha` specific tools. @vyasr do these scripts need to be self contained, and we set actions related variables outside the script?

Reply:
If we want to move towards being able to run the steps for doing things locally then it's useful if the tools don't reference github-specific stuff

Reply:
Sounds good, in that case perhaps we should just export these two variables?